src/handlers/dns/DnsStreamHandler.h

```diff
@@ -241,6 +241,13 @@ class DnsStreamHandler final : public visor::StreamMetricsHandler<DnsMetricsMana
 {
     static constexpr size_t DNSTAP_TYPE_SIZE = 15;
 
+    struct DnsCacheData {
+        uint32_t flowKey = 0;
```

Comment:
i don't think we should use flowKey at all - after consideration, it's possible to receive two different packets with the same flowKey that need should not reuse cache. timestamp should be more accurate and sufficient

Reply:
As I mentioned. using only packet timestamp still not unique enough. IT happens to have packets with same timestamp at least in pktvisor tests. So I need to use both FlowKey AND timestamp (sec and nsec)

Reply:
only timestamp breaks pktvisor tests: i.e. get different results from what currently is

Reply:
we should fix those tests then - i don't see how it's possible to legitimately get a different packet with the exact same time stamp. we'll need to dig in - but that's a better thing to fix then to try to use flowKey